src/olympia/editors/tests/test_models.py

```diff
@@ -399,7 +399,7 @@ def test_authors(self):
         AddonUser.objects.create(addon=addon, user=bert)
         AddonUser.objects.create(addon=addon, user=ernie)
         row = self.Queue.objects.all()[0]
-        assert row.authors == [(bert.id, 'bert'), (ernie.id, 'ernie')]
+        assert row.authors == [(ernie.id, 'ernie'), (bert.id, 'bert')]
```

Comment:
Oh, didn't see this before I merged.  I have some other issues on this code to work on so I'll change it then.